recipe/meta.yaml

```
@@ -27,6 +27,7 @@ build:
   # xref: https://github.com/conda-forge/cuda-feedstock/issues/27
   skip: true  # [ppc64le]
   script:
+    - pushd cuda
```

Hmmm I think this should be
```suggestion
    - pushd cuda_bindings
```

---

[In 12.6.1]( https://github.com/NVIDIA/cuda-python/tree/v12.6.1 ), this directory appears to be absent. Only the `cuda` directory is present

---

Thanks, John, for helping out. Didn't mean to leave it to you to do the work, I was distracted by other things while waiting for the CI to prove/disprove.

I think what happened was that we tagged first, and then change the repo layout later, so `cuda` is right here but will become `cuda_bindings` in the next release.

---

Yes this is part of the reason I need to rely more on your team to do the updates here. Am not tracking these layout changes and other packaging plans as closely. So this sort of thing is best left in your team's hands

Came to conclusion that this was the directory name due to CI failing on a simple version update and looking upstream to see how the tagged structure looked. Fortunately it was simple in this case